Changed the README (#2)
gradle/solutions-gradle-build-onboarding: README.md

> # Twitch4j onboarding project

Author: Ah yes, Istarted writing it bewfore we decided to rename it 👍

> The purpose of this project is to ease onboarding of new members to Gradle Enterprise. We've set this up to have all of 

Reviewer: ```suggestion
The purpose of this project is to learn how to improve Gradle builds with Gradle Enterprise. We've set this up to have 
```

> the commonly encountered build caching issues when helping customers improve their builds. The full list of caching issues

Reviewer: ```suggestion
 commonly encountered build issues. The full list of build issues
```

this project has is written below.

There are two branches used for the onboarding: `master` and `onboarding-broken-build`. `master` has most of the problems,
while `onboarding-broken-build` has some issues that actually break the build, so that's the reason for it being in a 
separate branch.

> **Do not push to this repository**, instead if you want to push your commits create a fork and do it there - 

Reviewer: once you have admin, can we disable pushing to master?
